api/admin.py

```diff
@@ -39,16 +40,65 @@ class PersonalInfoAdmin(admin.ModelAdmin[models.PersonalInfo]):
     form = PersonalInfoAdminForm
 
 
-admin.site.register(models.PersonalInfo, PersonalInfoAdmin)
+class StudentAdminForm(forms.ModelForm[models.Student]):
+    class Meta:
+        model = models.Student
+        fields = "__all__"
+
+
+class StudentAdmin(admin.ModelAdmin[models.Student]):
+    form = StudentAdminForm
+    readonly_fields = (
+        "enrollment_tests_summary",
+        "enrollment_tests_result_answers",
+    )
+
+    # TODO: searching is case-sensitive, better to make it case-insensitive somehow
```

Comment:
As far as I remember, we automatically normalize names and emails. That would mean the we don't have to make the search case-insensitive.

Reply:
Yes we do, but it doesn't make search easier. 
For example: we have student Ivan Petrov - you can't find him with text "ivan", only "Ivan", as it's stored in database.